insights/parsers/etc_libvirt_qemu_xml.py

```diff
@@ -0,0 +1,157 @@
+"""
+EtcLibvirtQemuXML file ``/etc/libvirt/qemu/*.xml``
+--------------------------------------------------
+"""
+from .. import XMLParser, parser
+from insights.specs import Specs
+
+
+@parser(Specs.etc_libvirt_qemu_xml)
+class EtcLibvirtQemuXML(XMLParser):
+    """This class parses xml files under ``/etc/libvirt/qemu/`` using
```

Comment:
@psachin what do you think about shortening this class name to `QemuXML` instead?

Reply:
I had the same opinion in mind, the long file and class name does not look so good but they clearly indicates the file path the parser will parse.

Reply:
Is it necessary to express the file path things in the class name and module name? This kind of name style is quite different from all the existing parsers. Unless the parser's logic depends on the specified `file_path`, I think it would be better to user QemuXML instead.

Reply:
class renamed to QemuXML